CMakeLists.txt

```diff
@@ -187,20 +187,12 @@ if( NOT DEFINED ENV{LOCAL_PATH_JEDI_TESTFILES} AND NOT DEFINED ${GIT_TAG_FUNC} )
   ecbuild_bundle( PROJECT ioda-data GIT "https://github.com/JCSDA-internal/ioda-data.git" BRANCH develop UPDATE )
 endif()
 
-# If IODA's current branch is available in ioda-data repo, that branch will be checked out
-branch_checkout (REPO_DIR_NAME ioda-data
-                 BRANCH ${GIT_BRANCH_FUNC} )
-
 # same procedure for ufo-data
 find_branch_name(REPO_DIR_NAME ufo)
```

Comment:
This line can be removed too, and in the line below, what does `${GIT_TAG_FUNC}` do/mean?
Same comment/question about ioda-data and fv3-jedi-data

Reply:
Yes, thanks. I left the local testfiles logic just in case we want to use that at some point, but removed the rest.